composer.json

```diff
@@ -34,6 +34,9 @@
         "behatch/contexts": "^2.6",
         "doctrine/doctrine-bundle": "^1.6.3",
         "doctrine/orm": "^2.5.2",
+        "doctrine/mongodb-odm": "^1.1.5",
+        "doctrine/mongodb-odm-bundle": "~3.0",
+        "ext-mongo": "*",
```

Comment:
Indeed, you need to [enable the extension in travis](https://docs.travis-ci.com/user/languages/php#PHP-extensions).

For example, add : 
```
before_install: 
 - echo "extension = mongodb.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```

also, it's `ext-mongodb` for php 7 not `ext-mongo`